index.html

```diff
@@ -48,8 +49,43 @@ <h2 class='no-margin-top'>
         {% endif %}
       </div>
     </div>
+      {% endfor %}
```

Comment:
this tag was initially between the divs on line 53 and 54, but I believe that was an error